internal/network/nsx_subresource.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 	validationutils "github.com/vmware/terraform-provider-vcf/internal/validation"
 	"github.com/vmware/vcf-sdk-go/models"
+	"strings"
```

Comment:
This will need to get resolved, as I've merged some commits that implement domain import feature.

Reply:
Resolved. https://github.com/vmware/terraform-provider-vcf/pull/59/commits/ae006843cd72abd63ee742669d7f9c2797cd1f33